lib/oli/rendering/report.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Oli.Rendering.Report do
+  @moduledoc """
+  This modules defines the rendering functionality for report.
+  """
+  import Oli.Utils
+
+  alias Oli.Rendering.Context
+
+  @callback report(%Context{}, %{}) :: [any()]
+  @callback error(%Context{}, %{}, {Atom.t(), String.t(), String.t()}) :: [any()]
+
```

Comment:
`Atom.t()` shows as an unknown type. Use `atom()` instead

```suggestion
  @callback error(%Context{}, %{}, {atom(), String.t(), String.t()}) :: [any()]
```

Reply:
done